src/drive/locales/en.json

```diff
@@ -643,5 +644,16 @@
       "text": "You're about to be redirected…",
       "error": "Error during the redirection. Generally, this means that the content of the file is not in the correct format."
     }
+  },
+  "Shortcut": {
+    "title_modal": "Create a shortcut",
+    "content_modal": "You can create a shortcut",
```

Comment:
I don't think this string is used, is it?

Reply:
You right, we removed it. Thanks!